virtual_rainforest/models/soil/soil_model.py

```diff
@@ -50,7 +51,9 @@ class SoilModel(BaseModel):
         data: The data object to be used in the model.
         start_time: A datetime64 value setting the start time of the model.
         update_interval: Time to wait between updates of the model state.
```

Comment:
I just noticed that the start_time is not in the `AbioticModel`, is it missing there or did we drop it altogether?

Reply:
It should be here actually, the docstring is out of date with the arguments of the `__init__`. `AbioticSimpleModel` does something with `start_time` (iirc), but models don't need it by default anymore. It being included here is a hangover from a while ago